css-canvas-medical/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+
+    <div class="row" id="nav-bar">
+      <div class="column-half">
+        <img src="images/logo-blue.svg" alt="canvas-blue">
+      </div>
+      <div class="column-half">
+        <a>Product</a>
+        <a>Implementation</a>
+        <a>About Us</a>
+        <a>Blog</a>
+        <input type="submit" value="Contact Us">
+      </div>
+    </div>
+
+    <div class="row" id="hero-portion">
+      <div class="hero-picture">
+        <div class="overlay"></div>
+        <section>
+          <h2 class="hero-portion-headline">The EMR Reimagined for<br> Primary Care</h2>
+          <p class="hero-portion-text">Cant remember the last time you talked with a patient without focusing<br>
+            on the EMR? Canvas let you rediscover joy in the work you do. Capture<br>
+            meaningful data, identify gaps in care, and move seamlessly between<br>
+            tasks within the platform. You stay present with patients while your<br>
+            whole care team stays up-to-speed. Be a better doctor with Canvas,<br>
+            designed for primary care providers with the help of primary care<br> providers.
+          </p>
+        </section>
+        <input type="submit" name="get-started" value="Get Started">
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-full">
+        <section class="clinicians-prefer">
+          <h2>Clinicians Prefer Canvas</h2>
+          <p>Physician-owned primary care practices across the country have<br> already
+            made Canvas the fabric of their care.
+          </p>
+        </section>
+      </div>
+    </div>
+
+    <div class="row" id="locations">
+      <div class="column-half">
+        <p class="left-column">Cascade Family Practice, Portland OR <br> Temescal Creek Medicine, Oakland CA<br>
+          San Juan Healthcare, Friday Harbor WA
+        </p>
```

Comment:
`br` elements should be avoided unless you have text that is all related that you want line breaks within. You should split this up into multiple `p` elements